app/controllers/collections_controller.rb

```diff
@@ -161,6 +161,21 @@ def delete_item_from_collection
     end
   end
 
+  def delete_document_from_item
```

Comment:
I'm not too sure how much of an issue this is, but the param names here don't match the names of params that were specified in the JIRA.